pkg/volume/local/local.go

```diff
@@ -208,6 +213,24 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return nil
 	}
 
+	oldMount, err := mount.IsAlreadyMounted(dir)
```

Comment:
I don't think we need to check if the volume is already mounted.  There's at least two scenarios where you still want to check fsgroup but this is the only mount.

1. The first time the volume is mounted.  The admin may have already created the volume with a specific fsgroup set.
2. Two pods running one after another, sharing the same volume.